core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java

```diff
@@ -73,7 +73,9 @@
 
   /** Conformance value that instructs Calcite to use SQL semantics
    * consistent with Microsoft SQL Server version 2008. */
-  SQL_SERVER_2008;
+  SQL_SERVER_2008,
+
+  PRESTO;
 
```

Comment:
PRESTO is not really a db, it behaves more like a computation engine, i'm not sure if we should add a new conformance for it.

Reply:
That's right. I am also debating on this. I could delete it or rename it. But I think the conformance is only for SQL string validation, so whether or not DB is kinda fine. But it's your call.

Reply:
Presto has its own SQL parser & validator & type system, so it counts as a 'dialect' (conformance) for these purposes.

Reply:
I'll add further comments to https://issues.apache.org/jira/browse/CALCITE-3789.